aroquev/src/main/webapp/index.html

```diff
@@ -50,47 +50,108 @@ <h3>Personal projects / Contests</h3>
+
+			<h3>Cycling</h3>
+			<p>I enjoy hitting the trail with my mountain bike! It is really cool being able to excercise surrounded by nature and expore new routes. By the way, I am currently also trying to get into road cycling.</p>
+			<p>Here are some pictures we have taken during our routes.</p>
+
+			<div class="slideshow-container">
+
+				<div class="mySlides fade">
+					<div class="numbertext">1 / 3</div>
+					<img src="images/huasteca.jpg" style="width:100%">
+					<div class="text">Huasteca Mountains</div>
+				</div>
+
+				<div class="mySlides fade">
+					<div class="numbertext">2 / 3</div>
+					<img src="images/rompepicos.jpg" style="width:100%">
+					<div class="text">Rompepicos Dam</div>
+				</div>
+
+				<div class="mySlides fade">
+					<div class="numbertext">3 / 3</div>
+					<img src="images/bosque.jpg" style="width:100%">
+					<div class="text">El Barrial</div>
+				</div>
+
+				<a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+				<a class="next" onclick="plusSlides(1)">&#10095;</a>
+        
+			</div>
+			<br>
+
+			<div style="text-align:center">
+				<span class="dot" onclick="currentSlide(1)"></span> 
+				<span class="dot" onclick="currentSlide(2)"></span> 
+				<span class="dot" onclick="currentSlide(3)"></span> 
+			</div>
+
+			<h3>Soccer</h3>
+			<p>I am a huge fan of soccer, and I enjoy watching and playing the sport (even on videogames!). I have always supported the local team Rayados, and thanks to my uncle I now also support 
+                German team Borussia Dortmund. Sometimes I think I could have been a professional soccer player!
+            </p>
+            <p>Here is a photo of my friend Chava and me at Rayados' stadium.</p>
+            <img class="body-img" src="images/stadium.jpg" alt="Chava and Armando at a soccer game">
+		</div>
+
 	  <p>Click here to get a random message:</p>
       
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+		<button onclick="addRandomGreeting()">Hello</button>
+		<div id="greeting-container"></div>
+
+
 
-    	
+    <div id="external-links">
+			<p><a id="github-link" href="https://github.com/aroquev00" target="_blank">Click here to see my GitHub Profile! <img src="images/github-logo.png"></a></p>
+      <p><a id="linkedin-link" href="https://mx.linkedin.com/in/armando-roque-a533a316b/en-us?trk=people-guest_people_search-card" target="_blank">Click here to see my LinkedIn Profile! <img src="images/Linkedin-logo.png"></a></p>
+		</div>
+        
 
     </div>
   </body>
   <script>calculateArmandoAge()</script>
+  <script>initSlides()</script>
```

Comment:
Can these calls be moved into script.js? This would require the script tag for script.js down to the bottom of the `body`

Reply:
Done.